src/map_main.cpp

```diff
@@ -307,22 +317,16 @@ int pandora_map(int argc, char *argv[]) {
 
     cout << now() << "Loading Index and LocalPRGs from file" << endl;
     auto index = std::make_shared<Index>();
-    index->load(prgfile, w, k);
+    index->load(prgfile, w, k); //load the index built in the index step
```

Comment:
is this comment necessary? It seems fairly self-explanatory to me.

Reply:
Done in https://github.com/leoisl/pandora/commit/5b329a78d89eaa0f85592258a8fb378677f71587